app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEvent.kt

```diff
@@ -41,4 +41,6 @@ sealed class BpPassportScanned : InstantSearchEvent() {
   data class ByPatientFound(val patientId: UUID) : BpPassportScanned()
 
   data class ByPatientNotFound(val identifier: Identifier) : BpPassportScanned()
+
+  data class ByShortCode(val shortCode: String) : BpPassportScanned()
```

Comment:
Resolved in earlier comments

Reply:
When BP passport is scanned we are using `ScanResult` to send back the status which is named properly. I haven't changed this naming convention so that it's consistent with what we have on the home screen.

Reply:
@vinaysshenoy why did we use the `By` naming convention here? As Janhavi pointed out I am not entirely sure if that makes sense. 🤔 

In this feature, I copied our original implementation naming convention to make it consistent. But if renaming makes sense, I will do that here.